examples/tutorials/04_linear_models_example.ipynb

```diff
@@ -21,20 +21,23 @@
    "outputs": [],
```

Comment:
<p>Fix import order and blank lines and combine plt.style</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>